Content.Server/BodySystem/BodyPart.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace Robust.Shared.BodySystem {
+
+
+    public partial class BodyPart {
```

Comment:
Please don't use a partial to share the same class for instantiated body parts and their prototypes. Prototypes should 100% be immutable global data and instantiated parts an entirely separate class.

You can just keep a reference to the `BodyPartPrototype` in the `BodyPart` if you need to access data like the name or whatever.